linux/ibus-keyman/src/main.c

```diff
@@ -51,7 +51,7 @@ ibus_disconnected_cb (IBusBus  *bus,
                       gpointer  user_data)
 {
     g_debug ("bus disconnected");
-    KeymanService *service = km_service_get_default();
+    KeymanService *service = km_service_get_default(NULL);
```

Comment:
Is this going to overwrite a previously set `engine` internally?

Reply:
Yes. `engine` will be set while a Keyman keyboard is active.